src/omotes_sdk/internal/worker/params_dict.py

```diff
@@ -51,6 +51,22 @@ def parse_workflow_config_parameter(
         # `workflow_config[field_key]` according to the type checker. However, we have confirmed
         # the type already so we may cast it.
         result = cast(ParamsDictValue, maybe_value)
+    elif is_present and type(maybe_value) is float and expected_type is int:
+        # when the field value type is float but the Expected type is an integer,
+        # round the value to an integer and log the warning message
+        # if the non-rounded value is received.
+        rounded_maybe_value = round(maybe_value)
+        result = cast(ParamsDictValue, rounded_maybe_value)
```

Comment:
Did you check if this cast is required? I would expect that `round`  always delivers an `int` and therefore the type of `rounded_maybe_value` is known.

Reply:
Initially, I added `result = round(maybe_value)` at line 58, this results in the following error message at the end. Therefore added `cast()` here
<img width="725" alt="image" src="https://github.com/Project-OMOTES/omotes-sdk-python/assets/168736045/f1f97665-a39e-406f-abfc-4dd96bf4da5c">